.vscode/launch.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch Program",
+      "program": "${workspaceFolder}/src/index.js",
+      "skipFiles": ["<node_internals>/**"]
```

Comment:
In a follow-up PR, it would be cool to add various launch.json options to debug the code too.

Reply:
I can file an issue for this